single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DirectionalFeederInfo.java

```diff
@@ -42,6 +43,12 @@ public DirectionalFeederInfo(String componentType, double value, DoubleFunction<
         this.value = value;
     }
 
+    public DirectionalFeederInfo(String componentType, double value, String unit, BiFunction<Double, String, String> formatter) {
```

Comment:
Missing `String userDefinedId` parameter
Add a new constructor as following : 
```java
public DirectionalFeederInfo(String componentType, double value, String unit, BiFunction<Double, String, String> formatter, String userDefinedId) {
        super(componentType, null, formatter.apply(value, unit), userDefinedId);
        this.arrowDirection = Objects.requireNonNull(getArrowDirection(value));
        this.value = value;
}
```
And change current constructor as following:
```java
public DirectionalFeederInfo(String componentType, double value, String unit, BiFunction<Double, String, String> formatter) {
       this(componentType, value, unit, formatter, null);
}
```